assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserDimensionsPieChart.js

```diff
@@ -142,6 +150,28 @@ export default function UserDimensionsPieChart( { dimensionName, dimensionValue,
 		}
 	}, [ chartID, setValues ] );
 
+	useEffect( () => {
+		const onTooltipClick = ( e ) => {
+			const { target } = e || {};
```

Comment:
One letter variables keep me up at night 👻 

```suggestion
		const onTooltipClick = ( event ) => {
			const { target } = event || {};
```